hbp_knowledge/receptors/djouma2006.bel

```diff
@@ -74,67 +74,117 @@ DEFINE ANNOTATION TextLocation  AS URL "https://arty.scai.fraunhofer.de/artifact
 
 SET Evidence ="Furthermore a significant interaction was found between housing and treatment in both the central nucleus of the amygdala (CeA; F1,139 1⁄4 11.560, P < 0.001; Fig. 3F) and basolateral amygdala (BLA; F1,137 1⁄4 7.616, P 1⁄4 0.007; Fig. 3E).Thus, isolation caused an up-regulation in dopamine D2 ⁄ 3 receptors in these regions, an effect that was reversed by antalarmin treatment (but only reached statistical significance in the CeA)."
-
+SET Confidence = "Medium"
+SET MeSHAnatomy = "Amygdala" #because the evidence indicates activities happening in amygdala
 path(HBP:"isolation rearing") -> p(RGD:Drd2)
 path(HBP:"isolation rearing") -> p(RGD:Drd3)
 a(MESH:antalarmin) -| path(HBP:"isolation rearing")
+UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence ="A significant decrease in BDNF mRNA expression resulting from ISO was found in the BLA ()25%; F1,102 1⁄4 77.937, P < 0.001), nucleus of the horizontal limb of the diagonal band (HB )75%; F1,67 1⁄4 259.106, P < 0.001) and dentate gyrus (DG )85%; F1,101 1⁄4 984.958,"
+SET Confidence = "Medium"
 SET MeSHAnatomy ={"Basolateral Nuclear Complex","Diagonal Band of Broca","Dentate Gyrus"}
 path(HBP:"isolation rearing") -| r(RGD:Bdnf)
+UNSET MeSHAnatomy #MeSHAnatomy was unset
+UNSET Confidence
 
 SET Evidence ="whilst isolation increased BDNF mRNA expression in the retrosplenial cortex"
-#SET MeSHAnatoomy =""
+SET Confidence = "Medium"
+SET Anatomy ="retrosplenial granular cortex"
 path(HBP:"isolation rearing") -> r(RGD:Bdnf)
+#MeSHAnatomy was set and unset
+UNSET Anatomy
+UNSET Confidence
 
 SET Evidence ="Treatment with antalarmin had no impact upon the isolation-induced alterations of BDNF expression."
+SET Confidence = "Medium"
+path(HBP:"isolation rearing") -> r(RGD:Bdnf)
+#above Statement was added because of "isolation-induced alterations of BDNF expression"
 a(MESH:antalarmin) cnc r(RGD:Bdnf)
 a(MESH:antalarmin) cnc path(HBP:"isolation rearing")
+UNSET Confidence
 
 SET Evidence ="ISO resulted in a decrease in TrkB mRNA expression in the cingulate cortex (Cg1–3; F1,124 1⁄4 7.582, P 1⁄4 0.014) and piriform cortex (Pir; F1,119 1⁄4 3.869"
+SET Confidence = "Medium"
 SET MeSHAnatomy ={"Gyrus Cinguli","Piriform Cortex"}
 path(HBP:"isolation rearing") -| r(RGD:Ntrk2)
+UNSET MeSHAnatomy
+#MeSHAnatomy was unset
+UNSET Confidence
 
 SET Evidence ="whereas an increase was observed in isolates compared to GH in the hippocampus (+38%; F1,59 1⁄4 14.131, P < 0.001), DG (+32%; F1,56 1⁄4 8.398, P 1⁄4 0.005) and RSc (+32%; F1,56 1⁄4 6.830, P 1⁄4 0.011). "
+SET Confidence = "Medium"
 SET MeSHAnatomy ={"Hippocampus","Dentate Gyrus"}
 path(HBP:"isolation rearing") -> r(RGD:Ntrk2)
+UNSET MeSHAnatomy
+#MeSHAnatomy was unset
+UNSET Confidence
 
 SET Evidence ="As was the case with BDNF expression, treatment with antalarmin had no impact upon the isolation-induced alterations in TrkB expression."
+SET Confidence = "Medium"
+path(HBP:"isolation rearing") -> r(RGD:Ntrk2)
+a(MESH:antalarmin) cnc r(RGD:Ntrk2)
```

Comment:
I would stick with proteinabundance, as the expression and not transcription is mentioned
```suggestion
a(MESH:antalarmin) cnc p(RGD:Ntrk2)
```